P/pocl/pocl@7/common.jl

```diff
@@ -162,8 +162,9 @@ function build_script(standalone=false)
         cp -va /opt/$target/lib/gcc/$target/*/*.{o,a} $prefix/share/lib
     elif [[ "${target}" == *-apple-darwin* ]]; then
         cp -va "$sysroot/usr/lib/crt1.o" "$prefix/share/lib/"
-        cp -va "$sysroot/usr/lib/libSystem.tbd" "$prefix/share/lib/"
-        cp -va "$sysroot/usr/lib/libm.tbd" "$prefix/share/lib/"
+        cp -va "$sysroot/usr/lib/libSystem.*tbd" "$prefix/share/lib/"
```

Comment:
For what is worth, you can use `-L` to follow symlinks.

Reply:
We want to keep the symlinks, though, to make `-lm` work (which has to find `libm.tbd` even if it's just pointing to `libSystem.tbd`).